File: Python Excel Examples/WorkbookApiDemo/getWorkbook.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

name = "GetWorkbook.xlsx"
storage = ""
password = ""
folder = "ExcelDocument"
result=api.get_workbook(name, password=password, storage=storage, folder=folder)

File: PHP Excel Examples/WorkbookApiDemo/getWorkbook.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorkbookApi($configuration);

#supports formats: Xlsx, Xlsb, Xls,xlsm,xltm,xltx, Ods
$name = "GetWorkbook.xlsx";
$password = null;
$storage = null;
$folder = "input";
$result = $apiInstance->getWorkbook($name, $password, $storage, $folder);